src/main/api/2/studio-api-2.yaml

```diff
@@ -3682,6 +3682,55 @@ paths:
         '500':
           $ref: '#/components/responses/InternalServerError'
 
+  /content/create_folder:
```

Comment:
@phuongnq please refer to the API v2 document. This would now be 
POST /content/{siteId}/folder

Reply:
I updated to new API endpoint. Thank you!